src/internal/deviceInfo.ts

```diff
@@ -23,6 +19,7 @@ export const deviceInfo = ((): {
   const isAndroid = /android/i.test(userAgent);
 
   return {
+    isIOS,
```

Comment:
Is `isIOS` used anywhere? I'm not sure why we would need to expose it.

Reply:
Ah no, indeed - I will remove it again